include/SFML/Window/WindowBase.hpp

```diff
@@ -604,14 +603,14 @@ class SFML_WINDOW_API WindowBase
 /// \class sf::WindowBase
 /// \ingroup window
 ///
-/// sf::WindowBase serves as the base class for all Windows.
+/// `sf::WindowBase` serves as the base class for all Windows.
 ///
 /// A sf::WindowBase can create its own new window, or be embedded into
```

Comment:
```suggestion
/// An `sf::WindowBase` can create its own new window, or be embedded into
```
Should we also audit the docs for the use of "a" prior to an `sf::` type? I think it reads better to as "an" as the article.